lib/ui/process_ui/demographic_details/age_date.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+
+class AgeDate extends StatefulWidget {
+  const AgeDate({super.key, required this.validation, required this.onChanged});
+
+  final RegExp validation;
+  final Function(String) onChanged;
+
+  @override
+  State<AgeDate> createState() => _AgeDateState();
+}
+
+class _AgeDateState extends State<AgeDate> {
```

Comment:
The date format need not be always in DD/MM/YYYY format. lets take the date format from UI SPEC, if nothing is provided then the default format will be DD/MM/YYYY